pkg/pillar/cmd/volumemgr/volumemgr.go

```diff
@@ -337,6 +338,7 @@ func Run(ps *pubsub.PubSub) {
 	populateInitialOldVolumeStatus(&ctx, rwImgDirname)
 	populateInitialOldVolumeStatus(&ctx, roContImgDirname)
 	populateExistingVolumesFormat(volumeEncryptedDirName)
+	populateExistingVolumesFormat(volumeClearDirName)
```

Comment:
I didn't realize these just create a map. Can't that map be found in the protobuf .go files instead of interating all the files in the directories?

Reply:
Sorry, I didn't understand the use of the map. Makes sense to clarify that with some comments (in another PR is fine.)